src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```
@@ -601,18 +654,33 @@ private static bool SupressesImport(MemberInfo member, Dictionary<MethodInfo, At
             if (null == member)
                 return true;
 
-            object[] atts = member.GetCustomAttributes(false);
+            var reflectionContext = member.Module.Assembly.ReflectionOnly;
+            object[] atts = null;
+            if (reflectionContext)
+            {
+                atts = member.AttributesFromReflectionContext();
+            }
+            else
+            {
+                atts = member.GetCustomAttributes(false);
+            }
 
             var method = member as MethodInfo;
             if (method != null)
             {
-                // Skip importing methods that have out and ref parameters
-                // as these are not supported currently: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-5460
-                ParameterInfo[] parameters = method.GetParameters();
-                foreach (var parameter in parameters)
+                try
+                {
+                    // Skip importing methods that have out and ref parameters
+                    // as these are not supported currently: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-5460
+                    ParameterInfo[] parameters = method.GetParameters();
+                    foreach (var parameter in parameters)
+                    {
+                        if (parameter.ParameterType.IsByRef == true)
+                            return true;
+                    }
+                }
+                catch (Exception)
```

same as above, empty catches don't need the exception